build/jobs/deploy-aks.yml

```diff
@@ -21,6 +21,9 @@ parameters:
 - name: schemaAutomaticUpdatesEnabled
   type: boolean
   default: false
+- name: reindexEnabled
+  type: boolean
+  default: false
```

Comment:
I don't think this is needed as a parameter. Is there a case where we would want to deploy to our test servers without reindex enabled?